docs/source/usage.rst

```diff
@@ -127,7 +130,7 @@ For hybrid peeling, where a large amount (millions of segregating sites) of sequ
 
 The ``-geno_error_prob``, ``-seq_error_prob`` and ``-rec_length`` arguments control some of the model parameters used in the model. ``-seq_error_prob`` must not be zero. |Software| is robust to deviations in genotyping error rate and sequencing error rate so it is not recommended to use these options unless large deviations from the default are known. Changing the ``-length`` argument to match the genetic map length can increase accuracy in some situations.
 
-The ``-est_geno_error_prob`` and ``-est_seq_error_prob`` options estimate the genotyping error rate and the sequencing error rate based on miss-match between observed and inferred states. This option is generally not necessary and can increase runtime. ``-est_alt_allele_prob`` estimates the alternative allele probabilities after each peeling cycle. This option can be useful if there are a large number of non-genotyped founders.
+The ``-est_geno_error_prob`` and ``-est_seq_error_prob`` options estimate the genotyping error rate and the sequencing error rate based on miss-match between observed and inferred states. This option is generally not necessary and can increase runtime. ``-est_alt_allele_prob`` estimates the alternative allele probabilities after each peeling cycle. This option can be useful if there are a large number of non-genotyped founders.  However, if ``-alt_allele_prob`` input is used, any ``-est_alt_allele_prob`` will be ignored.
```

Comment:
@RosCraddock if both are used, then we use `alt_allele_prob` as a starting point for estimation triggered by `est_...`. Please revise

Reply:
@gregorgorjanc Are we going to add some updates for the `alt_allele_prob` later in the peeling cycle as well? Currently, we only have one estimation at the very beginning of the peeling.  (line 19 in `tinypeel.py`)

Reply:
I am aware of this. Let's worry about the estimation step later as part of #148